tasks/configure.yml

```diff
@@ -5,16 +5,15 @@
 - set_fact: archivematica_src_configure_ss_api_key={{ 999999999999999999999 | random | to_uuid | hash('md5') }}
   when: "archivematica_src_configure_ss_api_key is undefined and archivematica_src_configure_ss|bool"
 
-# Delete default SS superuser test
-- name: "Delete default SS superuser"
+# Check whether SS user exists
+- name:  Check SS user
   become: "yes"
-  ignore_errors: "yes"
-  shell: echo "from django.contrib.auth.models import User;User.objects.get(username='test').delete()" | {{ archivematica_src_ss_virtualenv }}/bin/python manage.py shell
-  args:
-    chdir: "{{ archviematica_src_ss_app }}"
-    executable: /bin/bash
-  environment: "{{ archivematica_src_ss_environment }}"
-  when: "archivematica_src_configure_ss|bool"
+  command: sqlite3  "{{archivematica_src_ss_environment['SS_DB_NAME']}}" "select username from auth_user where username='{{ archivematica_src_configure_ss_user }}';"
```

Comment:
Does this mean that SS 0.11 won't get configured? (The "test" user is created as part of migrations)